test/spec/modules/userId_spec.js

```diff
@@ -1434,6 +1438,30 @@ describe('User ID', function () {
       }, {adUnits});
     });
 
+    it('test hook from floc cookies', function (done) {
+      // simulate existing browser local storage values
+      coreStorage.setCookie('flocId', JSON.stringify({id: 'Sample_Cohort', version: 'chrome1.1'}), (new Date(Date.now() + 5000).toUTCString()));
+
+      setSubmoduleRegistry([flocIdSubmodule]);
+      init(config);
+      config.setConfig(getConfigMock(['originFloc', 'flocId', 'cookie']));
+
+      requestBidsHook(function () {
+        adUnits.forEach(unit => {
+          unit.bids.forEach(bid => {
+            expect(bid).to.have.deep.nested.property('userId.flocId');
+            expect(bid.userId.flocId).to.deep.equal({id: 'Sample_Cohort', version: 'chrome1.1'});
+            expect(bid.userIdAsEids[0]).to.deep.equal({
+              source: 'chrome.com',
+              uids: [{id: 'Sample_Cohort', atype: 1, ext: { ver: 'chrome1.1' }}]
+            });
+          });
+        });
+        coreStorage.setCookie('flocId', '', EXPIRED_COOKIE_DATE);
+        done();
+      }, {adUnits});
+    });
```

Comment:
is this really a useful test? the module doesn't read from cookies and doesn't allow setting storage param (which the `getConfigMock()` is doing). I think you need to mock the `document.interestCohort()` instead